imagedephi.py

```diff
@@ -1,36 +1,37 @@
 import argparse
-import sys
 from enum import Enum
 from pathlib import Path
+import sys
+from typing import Dict, List
 
 import tifftools
-from tifftools import Datatype
+from tifftools import Datatype, TiffTag
 
 
 class RedactMethod(Enum):
     REPLACE = 1
     DELETE = 2
 
 
-def get_tags_to_redact():
+def get_tags_to_redact() -> Dict[int, Dict]:
```

Comment:
FYI, eventually it might help to turn this tag specification into a Data Class, to make is more structured than just a dict, but I think that's out of scope of this PR.